app/models/web_hook_event_type.rb

```diff
@@ -19,6 +19,15 @@ class WebHookEventType < ActiveRecord::Base
   default_scope { order('id ASC') }
 
   validates :name, presence: true, uniqueness: true
+
+  def self.all_excluding_disabled_plugin_web_hooks
```

Comment:
I love descriptive names but I think this one went over the top 😉 

What about `WebHookEventType.active` instead? (or something similar?)